test/sai_test/config/route_configer.py

```diff
@@ -739,6 +749,31 @@ def create_nexthop_by_rif(self, rif, nexthop_device: Device):
 
         return nhopv4, nhopv6
 
+    def create_nexthop_group_by_nexthops(self, dest_device: Device):
+        """
+        Create nexthop group by nexthops.
+        """
+        self.test_obj.nhop_group1 = sai_thrift_create_next_hop_group(
+            self.test_obj.client, type=SAI_NEXT_HOP_GROUP_TYPE_ECMP)
+
+        self.test_obj.nh_group_member1 = sai_thrift_create_next_hop_group_member(
+            self.test_obj.client,
+            next_hop_group_id=self.test_obj.nhop_group1,
+            next_hop_id=self.test_obj.dut.lag2.nexthopv4.nexthop_id)
+
+        self.test_obj.nh_group_member1 = sai_thrift_create_next_hop_group_member(
+            self.test_obj.client,
+            next_hop_group_id=self.test_obj.nhop_group1,
+            next_hop_id=self.test_obj.dut.lag3.nexthopv4.nexthop_id)
+
+        self.test_obj.ecmp_route0 = sai_thrift_route_entry_t(
```

Comment:
you can create a list to save nexthop groups
after create the nexthop group, it should be returned as other method